diff --git a/drivers/staging/qcacld-3.0/core/mac/src/sys/common/src/wlan_qct_sys.c b/drivers/staging/qcacld-3.0/core/mac/src/sys/common/src/wlan_qct_sys.c
--- a/drivers/staging/qcacld-3.0/core/mac/src/sys/common/src/wlan_qct_sys.c
+++ b/drivers/staging/qcacld-3.0/core/mac/src/sys/common/src/wlan_qct_sys.c
@@ -56,20 +56,13 @@ QDF_STATUS sys_build_message_header(SYS_MSG_ID msg_id,
  *
  * Return: none
  */
-#ifdef QDF_ENABLE_TRACING
 static void umac_stop_complete_cb(void *user_data)
 {
 	qdf_event_t *stop_evt = (qdf_event_t *) user_data;
 	QDF_STATUS qdf_status = qdf_event_set(stop_evt);
 
 	QDF_ASSERT(QDF_IS_STATUS_SUCCESS(qdf_status));
 }
-#else
-static void umac_stop_complete_cb(void *user_data)
-{
-	return;
-}
-#endif
 
 static inline QDF_STATUS umac_stop_flush_cb(struct scheduler_msg *msg)
 {
